app/models/merchant.rb

```diff
@@ -1,6 +1,8 @@
 class Merchant < ApplicationRecord 
   has_many :order_items
   has_many :products
+  has_many :orders
+
```

Comment:
Not sure if we connected this relation. The team can discuss on this Monday. If so, I think we will need to run a migration.

Reply:
I think this needs to be one of those distinct associations -- merchant has many orders through order_items, something along those lines.